src/pal/src/file/path.cpp

```diff
@@ -115,30 +112,29 @@ GetFullPathNameA(
             goto done;
         }
         
-        if (strcat_s(lpUnixPath, max_len, "/") != SAFECRT_SUCCESS)
+        if (!lpUnixPath.Append("/",1) ||
+            !lpUnixPath.Append(lpFileName,strlen(lpFileName))
+           )
         {
-            ERROR("strcat_s failed!\n");
+            ERROR("Append failed!\n");
             SetLastError(ERROR_FILENAME_EXCED_RANGE);
```

Comment:
The error code is no longer correct - it should return ERROR_NOT_ENOUGH_MEMORY now